### DISCUSSION TITLE
Can scripts be edited?

### DISCUSSION THREAD

**Discussion by HyunWinter** [thijsvanloef/palworld-server-docker#430] [Q&A] [answered]
Hi, I'm trying to localize the messages broadcasted in-game. I tried editing the script files but have had no luck so far. It seems like they are not used dynamically by docker build, or I'm missing something. Does anyone have an idea?

OS: Linux
File location: `scripts/auto_reboot.sh`
Code: (line 19)
`rcon-cli -c /home/steam/server/rcon.yaml "broadcast The_Server_will_NOT_reboot_in_${i}_Minutes"`

**Comment by win5923** [comment, marked as answer]
HI @HyunWinter, I added the command you mentioned to line 19, haven't encountered the issue you mentioned. Please check if the 'image' in the `docker-compose.yaml` corresponds to the image name after you've built the Docker image. Also, make sure to set `AUTO_REBOOT_ENABLED=true` and `AUTO_REBOOT_CRON_EXPRESSION`.

![image](https://github.com/thijsvanloef/palworld-server-docker/assets/56353753/fe5ad308-5481-4df0-bfac-0cfb06a5fd8c)

What version of the image are you using?

```
time="2024-02-22T17:36:00Z" level=info msg=starting iteration=0 job.command="bash /home/steam/server/auto_reboot.sh" job.position=1 job.schedule="1/5 * * * *"
time="2024-02-22T17:36:00Z" level=info msg="Broadcasted: The_Server_will_NOT_reboot_in_3_Minutes" channel=stdout iteration=0 job.command="bash /home/steam/server/auto_reboot.sh" job.position=1 job.schedule="1/5 * * * *"
time="2024-02-22T17:36:00Z" level=info msg="Complete Save" channel=stdout iteration=0 job.command="bash /home/steam/server/auto_reboot.sh" job.position=1 job.schedule="1/5 * * * *"
time="2024-02-22T17:36:00Z" level=info msg=Shutdown... channel=stdout iteration=0 job.command="bash /home/steam/server/auto_reboot.sh" job.position=1 job.schedule="1/5 * * * *"
time="2024-02-22T17:36:00Z" level=info msg="job succeeded" iteration=0 job.command="bash /home/steam/server/auto_reboot.sh" job.position=1 job.schedule="1/5 * * * *"
```

**Reply by HyunWinter** [reply]
Thank you so much for verifying. I was actually using an incorrect Docker image, and now I see the changes. It turns out non-English characters don't get transmitted properly, but this seems to be an issue on the RCON or recon-cli side.

**Comment by win5923** [comment]
> Thank you so much for verifying. I was actually using an incorrect Docker image, and now I see the changes. It turns out non-English characters don't get transmitted properly, but this seems to be an issue on the RCON or recon-cli side.

Yes, currently RCON does not support non-ASCII characters. This is a problem that Palworld devs need to address.

If you need a visual tool, I recommend [palworld-server-tool](https://github.com/zaigie/palworld-server-tool)
